helm/prometheus-agent-app/values.yaml

```diff
@@ -0,0 +1,15 @@
+remoteWrite:
+ - url: http://thanos-receive-ingestor-default.thanos.svc:19291/api/v1/receive
+
+labelSelector:
```

Comment:
I would have prefered a way to configure the whole value like https://github.com/giantswarm/prometheus-operator-app/blob/70ce9f1de0e565e6e412289483c17d2ca4283c01/helm/prometheus-operator-app/values.yaml#L2210 same for the namespace :)